lms/templates/discovery/course_card.underscore

```diff
@@ -12,7 +12,12 @@
                 <span class="course-code"><%= content.number %></span>
                 <span class="course-title"><%= content.display_name %></span>
             </h2>
-            <div class="course-date" aria-hidden="true"><%= interpolate(gettext("Starts: %s"), [start]) %></div>
+            <div class="course-date" aria-hidden="true">
+                <%= interpolate(
+                      gettext("Starts: %s"),
```

Comment:
Since you're here, can you please change this to use a better variable, eg `%(start_date)s`?